### PR TITLE
Added poster files.

### PR DESCRIPTION
**Pull request by impguard**
Yea. Poster files. Feel free to code review the AI file ;)

### CODE REVIEW

File: poster/icons/angular.ai [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
�8"=���ɴ.b�x��t���_��<q� �ǰPCO�^6
�����Ś�����^K�
�=���a(c��{�V#HK�c=�˯�]��ٯ��(�(��e/�ZJ!#�
�F\� Q��B��-ԤRB��+�Li�Ć@�FR�(V�\��Ơ��"��!�	���cBĠ!��g�햵}��s�f��o,o)�y2�ɘ�2ul�G���|_���c��=�t�c?fz
a.�B�I��M�u,�z�v�j=��)��{^À��a�����"!J�ӫQ�7PJ��u��N��=�6Nr�3q�w�xB#�����j����A\�vᬙ���Ċ���_�g�ml���
�&F6�G�������Ǉ��<� �57b�?������5�
��l�4� n%a!�y�Y�l��?����`�c1��܃�t�>�]��j?�@;�}�qt�(��v���j��9�*�ۡ]�Q���㫓5��6HI����
�
&wQ����X�?�_��o)��W���ʼ�>����4��v�����XM<�ҟ�j�2*�?��|v"�~�Y� �fs*u?ʍܻu�gʌz�l4����oC�Q���cv�
`7c��c��}ϸ�ڶ��!t�^��X
```
Comment (impguard) [inline poster/icons/angular.ai]: I'm not sure this is correct.

File: poster/icons/angular.ai [hunk 2]
```diff
@@ -0,0 +1,532 @@
...
+%99CC6699CC9999CCCC99CCFF99FF0099FF3399FF6699FF9999FFCC99FFFF
+%CC0000CC0033CC0066CC0099CC00CCCC00FFCC3300CC3333CC3366CC3399
+%CC33CCCC33FFCC6600CC6633CC6666CC6699CC66CCCC66FFCC9900CC9933
+%CC9966CC9999CC99CCCC99FFCCCC00CCCC33CCCC66CCCC99CCCCCCCCCCFF
+%CCFF00CCFF33CCFF66CCFF99CCFFCCCCFFFFFF0033FF0066FF0099FF00CC
+%FF3300FF3333FF3366FF3399FF33CCFF33FFFF6600FF6633FF6666FF6699
+%FF66CCFF66FFFF9900FF9933FF9966FF9999FF99CCFF99FFFFCC00FFCC33
+%FFCC66FFCC99FFCCCCFFCCFFFFFF33FFFF66FFFF99FFFFCC110000001100
+%000011111111220000002200000022222222440000004400000044444444
+%550000005500000055555555770000007700000077777777880000008800
```
Comment (nickfong) [inline poster/icons/angular.ai]: I don't think this is right.  This should be part of a `55500`, not a `55000`.  Please fix before merging.

**Comment by Fong-** [comment]
+1

**Comment by Fong-** [comment]
-1 Kill this branch and re-create with only the .pdf version of the poster

**Comment by nickfong** [comment]
+1

**Comment by impguard** [comment]
+1

**Comment by nickfong** [comment]
@ImpGuard You can't +1 your own work :P

Lucas +1ed in chat.  Merging in.

**Pull request merged**